apps/client/src/components/Form.tsx

```diff
@@ -1,23 +1,27 @@
 import React from "react";
-import { useHistory } from "react-router-dom";
 
 import { StyledForm } from "./FormStyles";
 
-const Form = ({ children, onSubmit, action, ...rest }) => {
-  const history = useHistory();
+export type FormProps = {
+  className?: string;
+  onSubmit?: Function;
+};
 
+const Form: React.FC<FormProps> = ({
+  children,
+  onSubmit,
+  ...otherProps
+  // otherProps could be: className, data-id or data-testid
+}) => {
   return (
     <StyledForm
       onSubmit={(e) => {
         e.preventDefault();
         if (onSubmit) {
           onSubmit(e);
         }
-        if (action) {
-          history.push(action);
-        }
       }}
-      {...rest}
+      {...otherProps}
```

Comment:
I prefer 'otherProps' > 'rest' lets do this uniform in the project. But maybe discuss in the dev meeting.